collect_app/src/main/java/org/odk/collect/android/activities/ViewFormHierarchyActivity.java

```diff
@@ -37,7 +38,11 @@
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        Collect.getInstance().getFormController().stepToOuterScreenEvent();
+        FormController formController = Collect.getInstance().getFormController();
+        if (formController == null) {
+            return;
+        }
```

Comment:
please add `else` clause and log the issue, the same reason like here https://github.com/opendatakit/collect/pull/2277#issuecomment-395383314

Reply:
Makes sense. I'll add logging in the parent class, as this should be the single point of checking for this error. Thanks for pointing that out.